packages/common/container-definitions/api-report/container-definitions.legacy.public.api.md

```diff
@@ -49,6 +49,4 @@ export interface ISelf {
     readonly clientId: string;
 }
 
-export { IThrottlingWarning }
```

Comment:
This one... the [type definition itself](https://github.com/microsoft/FluidFramework/blob/main/packages/common/core-interfaces/src/error.ts#L122-L127) (note, in `@fluidframework/core-interfaces`) is `@legacy + @alpha`. But [when we re-export it from container-definitions, we provide new TSDoc](https://github.com/microsoft/FluidFramework/blob/main/packages/common/container-definitions/src/index.ts#L76-L79) that only specify `@deprecated`... is that overriding the original tags and making it disappear?